modules/calib3d/test/test_calibration_hand_eye.cpp

```diff
@@ -427,9 +427,9 @@ class CV_CalibrateHandEyeTest : public cvtest::BaseTest
         eps_rvec_noise[CALIB_HAND_EYE_ANDREFF] = 1.0e-2;
         eps_rvec_noise[CALIB_HAND_EYE_DANIILIDIS] = 1.0e-2;
 
-        eps_tvec_noise[CALIB_HAND_EYE_TSAI] = 5.0e-2;
-        eps_tvec_noise[CALIB_HAND_EYE_PARK] = 5.0e-2;
-        eps_tvec_noise[CALIB_HAND_EYE_HORAUD] = 5.0e-2;
+        eps_tvec_noise[CALIB_HAND_EYE_TSAI] = 7.0e-2;
+        eps_tvec_noise[CALIB_HAND_EYE_PARK] = 7.0e-2;
+        eps_tvec_noise[CALIB_HAND_EYE_HORAUD] = 7.0e-2;
```

Comment:
Updated checks tolerance values to due to RNG generator update (stable and independent values: #16189)

/cc @catree